flake.nix

```diff
@@ -2,27 +2,41 @@
   description = "Nix derivations for PureScript core language tools.";
 
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/release-23.05";
+    nixpkgs.url = "github:nixos/nixpkgs/release-23.11";
 
     slimlock.url = "github:thomashoneyman/slimlock";
     slimlock.inputs.nixpkgs.follows = "nixpkgs";
 
     flake-compat.url = "github:edolstra/flake-compat";
     flake-compat.flake = false;
+
+    spago = {
+      url = "github:deemp/spago/add-flake-bk";
+      flake = false;
+    };
   };
 
-  outputs = {
+  outputs = inputs@{
```

Comment:
It can be before or after arguments, according to docs. Anyway, https://github.com/thomashoneyman/purescript-overlay/commit/6d5c9732e6d2edf1f7771b5c06e8e8a0c93b29b1 doesn't introduce it anymore